.rubocop.yml

```diff
@@ -32,7 +32,7 @@ Metrics/MethodLength:
   # There's ~2 long methods in `StripeClient`. If we want to truncate those a
   # little, we could move this to be closer to ~30 (but the default of 10 is
   # probably too short).
-  Max: 55
+  Max: 50
```

Comment:
I might've gotten it a bit shorter, but it turns out there's one other method that's also quite long (and it also doesn't have many obvious opportunities for shortening), so it wouldn't have helped that much.